src/data/minors/ds.ts

```diff
@@ -0,0 +1,328 @@
+            'BEE 4310',
+            'AEM 2010',
+            'AEM 2011',
+            'AEM 2820',
+            'AEM 4120',
+            'HD 4760',
+            'ASTRO 3334',
+            'ASTRO 3340',
+            'ASTRO 4523',
+            'CS 4780',
+            'CS 4787',
+            'CS 4850',
+            'ECE 4110',
+            'ECE 4520',
+            'ENGRD 2720',
+            'HADM 4010',
+            'HD 2930',
+            'HD 2940',
+            'INFO 2950',
+            'INFO 3300',
+            'INFO 3950',
+            'ORIE 4580',
+            'ORIE 4741',
+            'ORIE 4820',
+            'PSYCH 4750',
+            'PSYCH 4760',
+            'STSCI 3200',
+            'STSCI 4740',
+            'STSCI 4780',
+          ],
+
+          [
+            'COMM 4242',
+            'DSOC 2120',
+            'DSOC 4060',
+            'ENGL 3778',
+            'GOVT 3999',
+            'INFO 1200',
+            'INFO 1260',
+            'INFO 3200',
+            'INFO 4240',
+            'INFO 4250',
+            'INFO 4270',
+            'INFO 4561',
+            'PAM 2070',
+            'PAM 4060',
+            'PAM 4230',
+            'PAM 4540',
+            'STS 3440',
+            'STS 3561',
+          ],
+
+          [
+            'COMM 3150',
+            'COMM 3350',
+            'COMM 3189',
+            'COMM 4200',
+            'COMM 4360',
+            'COMM 4860',
+            'INFO 4310',
+          ],
+          [
+            'AEM 2770',
+            'AEM 3100',
+            'AEM 4060',
+            'AEM 4110',
+            'AEM 4225',
+            'AEM 4435',
+            'AEM 4620',
+            'AEM 4660',
+            'ASTRO 3310',
+            'BIOEE 3611',
+            'BIOEE 3620',
+            'BIOEE 4940',
+            'SOC 3580',
+            'BIOMG 4810',
+            'BIONB 3300',
+            'BIOMG 4870',
+            'BIONB 4220',
+            'BIONB 4380',
+            'BTRY 4381',
+            'BTRY 4820',
+            'BTRY 4830',
+            'BTRY 4840',
+            'CHEM 4810',
+            'CS 4740',
+            'DSOC 3140',
+            'EAS 3540',
+            'ECON 3120',
+            'ECON 3130',
+            'ECON 3140',
+            'ECON 4110',
+            'ENTOM 3030',
+            'HADM 4050',
+            'HADM 4770',
+            'INFO 3350',
+            'INFO 4100',
+            'ILRHR 4664',
+            'ILROB 4710',
+            'NS 4300',
+            'NTRES 3100',
+            'NTRES 3500',
+            'NTRES 4100',
+            'ORIE 2380',
+            'ORIE 4120',
+            'ORIE 4132',
+            'ORIE 4630',
+            'ORIE 4742',
+            'PAM 3280',
+            'PAM 3400',
+            'PAM 3600',
+            'PAM 3670',
+            'PAM 3780',
+            'PAM 3850',
+            'PAM 4080',
+            'PAM 4101',
+            'PAM 4110',
+            'PLSCS 4110',
+            'PLSCS 4200',
+            'PLSCS 4650',
+            'STS 4040',
```

Comment:
Does this minor only accept these courses? Or we can we check by ensuring that the course is of a certain level (e.g 3000+) using notation like `3***`.

Reply:
It seems as though the DS minor does indeed list all of these courses in excruciating detail.